<launch>
  <arg
    name="model" default="$(find hsr_br606)/urdf/hsr_br606.urdf"/>
  <arg
    name="gui"
    default="true" />
  <arg
	name = "sim"
	default="true" />

  <include file="$(find ur5_moveit_config)/launch/planning_context.launch" />

  <arg name="rvizconfig" default="$(find urdf_tutorial)/rviz/urdf.rviz" />

  <param name="robot_description" command="$(find xacro)/xacro.py $(find ur_description)/urdf/ur5_robot.urdf.xacro" />

  <param
    name="use_gui"
    value="$(arg gui)" />

  <param
    name="impedance_group"
    value="manipulator" />

  <param
    name="impedance_end_link"
    value="ee_link" />

  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="state_publisher" />

  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(arg rvizconfig)" required="true"/>

  <node
	name="impedance"
	pkg="hsr_impedance"
	type="hsr_impedance"
	args="$(arg sim)" />
</launch>
